ca_gazebo/launch/create_apriltag.launch

```diff
@@ -0,0 +1,27 @@
+<launch>
+	<arg name="env" default="apriltags"/>
+  	<arg name="paused" value="$(optenv PAUSED false)"/>
+	<arg name="gui" value="$(optenv GUI true)"/>
+	<arg name="debug" value="$(optenv DEBUG false)"/>
+	<env name="GAZEBO_MODEL_PATH" value="$(find ca_gazebo)/models"/>
```

Comment:
Don't include this here. First, check if this is necessary. If so, see [here](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_gazebo/package.xml#L24-L28).

Reply:
Adressed